dev/com.ibm.websphere.appserver.features/visibility/public/enterpriseBeansPersistentTimer-4.0/resources/l10n/io.openliberty.enterpriseBeansPersistentTimer-4.0.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2015 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature enables the use of persistent timers in Enterprise JavaBeans.
```

Comment:
Should we still be using Enterprise JavaBeans here in the description?

Reply:
Nice catch; thanks.  I've updated all the translations to use Jakarta Enterprise Beans and also fixed the copyright.